### DISCUSSION TITLE
Moveable stuff and house inventory

### DISCUSSION THREAD

**Discussion by Spreadcat** [hay-kot/homebox#829] [General]
Hi,

I am considering using homebox more and more when a though hit me:

Structuring the locations by floors/rooms will work fine. But one of the example in the demo was e.g. a TV in a living area. Considering that the location of the TV is secondary for me and more information like warranty, costs and vendor would be more important there is a drawback to this organization.

The second you move into a new house/appartment/living-space, all the locations are wrong while most like the rest of the information is still valid.

Organization wise it seems to make more sense to split items into at least two categories:

1. Stuff that is attached to the house and will not follow you
2. Stuff that likely will follow you when moving.

I therefore consider doing exactly that in homebox. One documentation related to the house with floors, where e.g. the lights, heating and kitchen appliances are listed and another category where I track items like TV, fridge, car, bike, tools, etc so that I don't have to update the location of all these items at any time.
I considered not putting them into Homebox at all, but that would require a new location to track this kind of information (or not track at all).

I wonder what other peoples thoughts are on this, do you even care or do you just accept to move all items into a new location in the case when you move?

**Comment by mingw128** [comment]
I don't think people move that often and I would think the location would generally stay the same. For instance, the blender in the kitchen in your old house is going to go in the kitchen in the new house, so a location of "kitchen" doesn't need to be updated at all. If you are someone that moves a lot and also rearranges where everything goes, then I would think that would be an outlier/edge-case.

**Reply by Spreadcat** [reply]
The question is: what is often? I live now in the third place with my better half and will certainly move at least one more time within the next years. These are just the places I owed and not even rented. The number there is in two digits for my other half. Luckily she's not using this documentation approach.
I see your point regarding stuff that is in the kitchen stays usually in the kitchen. But I see a difference there between e.g. installed appliances and things like a blender that would follow me. In case of moving that will lead to either deleting/archiving items and renaming locations or moving them to a new location I think.
I am still trying to get a feeling for how much work this will result in and a filling my instance with worthy items, though.
But it could be that I am discussing an edge-case here, ofc.
